ophyd/utils/__init__.py

```diff
@@ -1,11 +1,10 @@
 # vi: ts=4 sw=4 sts=4 expandtab
-import logging
+import logging  # noqa: F401
```

Comment:
Strange flake8 complained about it was imported, but unused. I am seeing it's used in line [9](https://github.com/bluesky/ophyd/pull/823/files#diff-f48b3a9fdaf7b4912ee2661cccd8805eR9) below.

Reply:
I guess in this case `noqa` is unnecessary.